### DISCUSSION TITLE
As a squad admin I want to see how many members are in the squad

### DISCUSSION THREAD

**Discussion by frank-connolly** [dailydotdev/daily#1454] [Feature Requests]
Unless I've missed it there is nowhere to see how many members my squad currently has. The (Just Java squad)[https://app.daily.dev/squads/justjava] just hit 3000 members but I only know to the nearest 100. Perhaps in the Squad settings there could be a place for the actual number of members (you could add other metadata here too but just an actual members number would be good).

**Comment by nimrodkra** [comment]
That's a great point, thank you! I'll take it to the team

**Comment by frank-connolly** [comment]
[like]  Frank Connolly reacted to your message:
________________________________
